markdown/beginners_guide.md

```diff
@@ -0,0 +1,61 @@
+# What is nf-core?
+A community effort to collect a curated set of analysis pipelines built using Nextflow.
+nf-core has three target audiences: facilities, single users and developers.
+For facilites it provides highly automated and optimized pipelines that guaranty reproducibility of results for their users.
+Single users profit from portable, documented and easy to use workflows. But you can also become a developer and create your own pipeline using already available templates.
+
+# What is Nextflow?
+Nextflow is a *workflow manager*.
+It has been developed specifically to ease the creation and execution of bioinformatics pipelines.
+The benefits of having you pipeline in Nextflow include:
+
+* Built in GitHub support.
+* Automated cluster execution.
```

Comment:
I don't think it's `Automated cluster execution`, but more like `Compatibility with most cluster executors`